### ISSUE TITLE
Run test data, got 0.48 c-index?!

### ISSUE THREAD

**Issue by yunkekeke** [jaredleekatzman/DeepSurv#12]
Hi there, 
I tested run the code but got a 0.48 c-index with the following code. Any idea why it could be the case? When printing out the log plot, the maximum likelihood is flat. Is there something going on with the loss function?

def generate_data(treatment_group = False):
    np.random.seed(123)
    sd = deepsurv.datasets.SimulatedData(5, num_features = 9,
        treatment_group = treatment_group)
    train_data = sd.generate_data(950)
    valid_data = sd.generate_data(300)
    test_data = sd.generate_data(300)

    return train_data, valid_data, test_data

train,valid,test = generate_data(True)

hyperparams = {
            'n_in': 10,
            'learning_rate': 1e-5,
            'hidden_layers_sizes': [10,10]
        }

network = DeepSurv(**hyperparams)

log = network.train(train,valid,n_epochs=500,validation_frequency = 1)

risk = network.predict_risk(test['x'])

**Comment by scienceaiden** [comment]
i got 0.4670 on test data- really low

**Comment by jaredleekatzman** [comment]
Some of the things you should look at:

- Normalize the data before training (and then normalize the training data by the same mean and variance of the training set)
- Did you try various learning_rates? and other hyper-parameters?

**Comment by scienceaiden** [comment]
data generated using generate_data(5000,2000,2000)

1. normalized data since theano has to have it
2 used optunity
still - c-index it does not show above 0.5

**Comment by jaredleekatzman** [comment]
@scienceML usually when you receive consistent 0.5 concordance index (or Nan) it is because your network is not training correctly. Usually this is due to problems in either: data normalization and preparation and hyper-parameter tuning. 
Did you perform a hyper-parameter search and then the network no longer works? Or are these results from a single test of hyper-parameters?

**Comment by scienceML** [comment]
@jaredleekatzman i prepared the dataset where all x values standardized (mean 0 var 1) , did not apply this logic to 'time' variable though.
